editor/state.js

```diff
@@ -34,6 +34,8 @@ export const editor = combineUndoableReducers( {
 					...state,
 					...action.edits,
 				};
+			case 'REQUEST_POST_UPDATE':
```

Comment:
Do you think it be more semantic for edits to clear themselves by a standalone action `CLEAR_POST_EDITS`, which the effect handler(s) for `REQUEST_POST_UPDATE` would be responsible for returning/dispatching? Still unsure just how much we want to tie editor state to network activity, especially now that we have patterns for relatively simple effect chains.